storage/cloud.py

```diff
@@ -1151,7 +1151,7 @@ def __init__(
         **kwargs,
     ):
         super(CloudFrontedS3Storage, self).__init__(
-            context, storage_path, s3_bucket, *args, **kwargs
+            context, storage_path, s3_bucket, s3_region=s3_region, *args, **kwargs
         )
 
         self.s3_region = s3_region
```

Comment:
Should we add a default region here? Like, `us-east-1`? Just for the sake of Quay.io, if nothing else.

Reply:
@syed is it possible this will break anything on Quay.io? I think since we use [this optimization](https://github.com/quay/quay/blob/master/storage/cloud.py#L1175-L1178) it would return the s3 endpoint with the region (`s3.us-east-1.amazonaws.com`) in the host name, not sure if that would break any existing users?

Reply:
it should be fine. We should be using the region specific endpoint anyways

Reply:
okay i ran a curl in a pod running in aws and it returned with the region in the hostname, so we should be good